src/validations/html/rules/MarkupOutsideContainer.js

```diff
@@ -0,0 +1,75 @@
+// This rule checks for any unexpected tags or text that
+// don't get caught by other validators, specifically:
+//
+// * Any tags or text outside the <html> element
+// * Any text that is inside the <html> element but not part
+//   of the <head> or <body> tags
+
+import findLastIndex from 'lodash-es/findLastIndex';
+
+import Code from './Code';
+
+export default class MismatchedTag {
```

Comment:
`TopLevelDocumentStructure`, `DocumentContainment`, etc.? Also be sure to name the class and the file the same

Reply:
Can discuss below, but for now I'm going with `NodeOutsideBody` because this is really an error about things that are only allowed in the body